OrbitGl/IntrospectionWindow.cpp

```diff
@@ -57,4 +57,13 @@ void IntrospectionWindow::ToggleRecording() {
   }
 }
 
+void IntrospectionWindow::RenderImGui() {
+  CaptureWindow::RenderImGui();
+
+  if (ImGui::CollapsingHeader("IntrospectionWindow")) {
+    IMGUI_VAR_TO_TEXT(IsIntrospecting());
+    IMGUI_VAR_TO_TEXT(time_graph_.GetTrackManager()->GetAllTracks().size());
```

Comment:
I would like to also have this Variable for CaptureInfo, WDYT?

Reply:
Yes good point, thanks, moved it over to CaptureWindow.